third_party/VENDOR-LICENSE

```diff
@@ -4058,6 +4091,33 @@ SOFTWARE.
 
 
 
+===========================================================
+Import: knative.dev/eventing/vendor/github.com/lightstep/tracecontext.go
+
+The MIT License (MIT)
+
+Copyright (c) 2016 
```

Comment:
<!--whitespace-bot-->
Remove trailing whitespace:
```suggestion
Copyright (c) 2016
```

Reply:
@mattmoor this file is code generated i suppose, can we exclude this linting?